app/database/itemStore.go

```diff
@@ -47,6 +84,38 @@ func (s *ItemStore) tableName() string {
 	return "items"
 }
 
+
+func (s *ItemStore) GetRawNavigationData(rootID, userID, userLanguageID, defaultLanguageID int64) (*[]NavigationItemChild, error){
+	var result []NavigationItemChild
+	if err := s.Raw(
+		"SELECT union_table.ID, union_table.sType, union_table.bTransparentFolder, " +
+			"COALESCE(union_table.idItemUnlocked, '')<>'' as hasUnlockedItems, " +
+			"COALESCE(ustrings.sTitle, dstrings.sTitle) AS sTitle, " +
+			"users_items.iScore AS iScore, users_items.bValidated AS bValidated, " +
+			"users_items.bFinished AS bFinished, users_items.bKeyObtained AS bKeyObtained, " +
+			"users_items.nbSubmissionsAttempts AS nbSubmissionsAttempts, " +
+			"users_items.sStartDate AS sStartDate, users_items.sValidationDate AS sValidationDate, " +
+			"users_items.sFinishDate AS sFinishDate, " +
+			"union_table.iChildOrder AS iChildOrder, " +
+			"union_table.idItemParent AS idItemParent " +
+			"FROM " +
+			"(SELECT items.*, NULL AS idItemParent, NULL AS iChildOrder FROM items WHERE items.ID=? UNION " +
+			"(SELECT items.*, idItemParent, iChildOrder FROM items, items_items " +
```

Comment:
I always prefer writing an explicit `X JOIN Y on JOIN_COND` than `FROM X, Y WHERE JOIN_COND`, don't you?... actually isn't the join condition missing here?

Reply:
@smadbe the join condition isn't missing here. It is ```WHERE items.ID=idItemChild ```. I have no preference about explicit/implicit styles. I'll make the JOIN explicit here.